wb_modbus_device_editor/template_manager.py

```diff
@@ -82,11 +82,12 @@ def get_parameter_enum(self, parameter_id) -> dict:
 
         return {"enum": enum, "enum_titles": enum_titles}
 
-    def calc_parameter_condition(self, condition, values):
 
+    def calc_parameter_condition(self, condition, values):
         try:
             condition = condition.replace("||", " or ").replace("&&", " and ")
-            return eval(condition, {"__builtins__": None}, values)
+            condition = re.sub(r"isDefined\(([A-Za-z1-9_]+)\)",r"('\g<1>' in locals())", condition)
+            return eval(condition, {"__builtins__": None, "locals":locals}, values)
```

Comment:
а через подстановку сюда `isDefined` не заработало?

ну в духе

```suggestion
            return eval(condition, {"__builtins__": None, "isDefined": lambda x: x in locals()}, values)
```

Reply:
Так работать не будет, потому что в лямбду попадает значение переменной,  а не имя
![image](https://github.com/wirenboard/wb-modbus-device-editor/assets/97599621/1baa2b0c-84a8-4cbe-a0f3-2d75867868a2)